socorro/external/boto/crashstorage.py

```diff
@@ -31,6 +35,13 @@ class BotoS3CrashStorage(CrashStorageBase):
 
     required_config = Namespace()
     required_config.add_option(
+        "resource_class",
+        default=ConnectionContext,
```

Comment:
Why does this get to be written like this when other options have to have their name written out as a string?